filebeat/fileset/fileset.go

```diff
@@ -33,6 +33,8 @@ import (
 	"strings"
 	"text/template"
 
+	"github.com/pkg/errors"
```

Comment:
This should be "errors" package. Is master also importing "github.com/pkg/errors"?